_layouts/team-page.html

```diff
@@ -13,7 +13,11 @@
         </div>
 
         <div class="col">
-            <h1 class="title">{{ page.firstname }} {{ page.lastname }}</h1>
+            <h1 class="title">{{ page.firstname }} {{ page.lastname }}
+                {% if page.pronouns %}
+                    <span class = "pronouns"> ({{page.pronouns}})</span>
```

Comment:
In my opinion, the smaller text here looks a bit out of place. I am likely not the one to consult on design here though.

Reply:
I think the small text is ok here, it follows design conventions on other sites!